hawkbit-dmf-amqp/src/test/java/org/eclipse/hawkbit/amqp/AmqpMessageHandlerServiceTest.java

```diff
@@ -313,11 +313,9 @@ public void authenticationRequestAllowedForArtifactWhichExistsAndAssignedToTarge
 
         // mock
         final LocalArtifact localArtifactMock = mock(LocalArtifact.class);
-        final Action actionMock = mock(Action.class);
         final DbArtifact dbArtifactMock = mock(DbArtifact.class);
         when(artifactManagementMock.findFirstLocalArtifactsBySHA1(anyString())).thenReturn(localArtifactMock);
-        when(controllerManagementMock.getActionForDownloadByTargetAndSoftwareModule(anyObject(), anyObject()))
-                .thenReturn(actionMock);
+        when(controllerManagementMock.hasTargetArtifactAssigned(anyObject(), anyObject())).thenReturn(true);
```

Comment:
Why you don't use `localArtifactMock` and the targetId used by the test?
So you only mock expected behavior and don't return anything by accident.